pyproject.toml

```diff
@@ -8,7 +8,10 @@ readme = "README.md"
 homepage = "https://github.com/m3dev/gokart"
 repository = "https://github.com/m3dev/gokart"
 documentation = "https://gokart.readthedocs.io/en/latest/"
-
+packages = [
+  {include = "gokart"},
+  {include = "gokart/py.typed"},
```

Comment:
@dn070017 [question]

`include = "gokart"` doesn't include `gokart/py.typed` ? 

I'm not sure but `include = "gokart"` sounds `include = "gokart/*"` which includes `gokart/py.typed` ...?

Reply:
@yokomotod 
Yes, I think you are correct. After investigate into the behavior for this setting, it turns out that Poetry includes every file under the source directory by default. I think it should be fine to revert the changes I made in `pyproject.toml`.

Reply:
👍 👍 👍